roles/jenkins/meta/main.yaml

```diff
@@ -1,16 +1,16 @@
 ---
 galaxy_info:
-  author: {{ cookiecutter.full_name }}
```

Comment:
I don't understand why you moved these templates?
You should have used the cookiecutter tool to generate a basic role.
Please fix it.

Reply:
I got files from https://github.com/geerlingguy/ansible-role-jenkins and https://github.com/geerlingguy/ansible-role-java ...
I left meta but changed molecule similar to common role